docs/docs/guides/architecture.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Architecture of the Airy Core Plaform
+sidebar_label: Architecture
+---
+
+## Overview
+
+The Airy Core Platform is a messaging platform comprised of a backend and frontend system.
+
+The `backend` system is a streaming platform and its role is to:
+
+- Ingest conversational events from different sources (mostly via webhook integrations), process them and store them in an Apache Kafka cluster
+- Once processed, the events are available and accessible through an [API](api/http.md)
+- Expose conversational events via a [webhook](api/webhook.md) integration
+
+Furthermore, the platform provides an `admin` application which manages authentication and authorization features.
```

Comment:
I would remove that line since it is confusing and already better explained further down under API

Reply:
Will rephrase it, because I want to say that it is not only about processing messages.